sys_siglist is not supported by default on uclibc and is, in general,
a bad thing for the same reasons as sys_errlist is a bad thing.  GNU
supports strsignal in the manner of strerror

--- alsa-utils-1.0.8/alsamixer/alsamixer.c.orig	2005-09-06 12:44:41.415219126 -0700
+++ alsa-utils-1.0.8/alsamixer/alsamixer.c	2005-09-06 12:44:43.407344482 -0700
@@ -2188,7 +2188,7 @@
 mixer_signal_handler (int signal)
 {
   if (signal != SIGSEGV)
-    mixer_abort (ERR_SIGNAL, sys_siglist[signal], 0);
+    mixer_abort (ERR_SIGNAL, strsignal(signal), 0);
   else
     {
       fprintf (stderr, "\nSegmentation fault.\n");
